[Term Entry] Java Date Method: .after() (#2703)
Codecademy/docs: content/java/concepts/date/terms/after/after.md

---
Title: '.after()'

> Description: 'Checks if the first date occurs after the second date.'

Reviewer: ```suggestion
Description: 'Checks if a date occurs after a specified date.'
```

Subjects:
  - 'Code Foundations'
  - 'Computer Science'
Tags:
  - 'Date'
  - 'Methods'
  - 'Booleans'
CatalogContent:
  - 'learn-java'
  - 'paths/computer-science'
---

> The **`Date.after()`** method Checks if the first date occurs after the second date. It returns a boolean value.

Reviewer 2: ```suggestion
The **`Date.after()`** method checks whether the first date occurs after the second date and returns a boolean value.
```

> The **`Date.after()`** method checks whether the first date occurs after the second date and returns a boolean value.

Reviewer: ```suggestion
The **`Date.after()`** method checks whether a date occurs after a specified date and returns a boolean value.
```
